pkg/rancher-desktop/pages/containers/ContainerInfo.vue

```diff
@@ -0,0 +1,385 @@
+          @click="clearSearch"
+        >
+          <i
+            aria-hidden="true"
+            class="icon icon-x"
+          />
+        </button>
+      </div>
+    </div>
+    <div class="tab-content">
+      <container-logs
+        v-if="containerId"
+        ref="containerLogs"
+        :container-id="containerId"
+        :is-container-running="isRunning"
+        :namespace="settings?.containers?.namespace"
+      />
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { ref, computed, watch, onMounted, onBeforeUnmount, nextTick } from 'vue';
+import { useRoute, useRouter } from 'vue-router';
+import { useStore } from 'vuex';
+
+import ContainerLogs from '@pkg/components/ContainerLogs.vue';
+import type { Container } from '@pkg/store/container-engine';
+import { ipcRenderer } from '@pkg/utils/ipcRenderer';
+
+// Router and Store
+const route = useRoute();
+const router = useRouter();
+const store = useStore();
+
+// Template refs with proper typing
+const containerLogs = ref<InstanceType<typeof ContainerLogs> | null>(null);
+const searchInput = ref<HTMLInputElement | null>(null);
+
+// Reactive data
+const activeTab = ref('logs');
+const settings = ref<any>();
+const subscribeTimer = ref<ReturnType<typeof setTimeout>>();
+const searchTerm = ref('');
+
+// Vuex integration
+const isK8sReady = computed(() => store.getters['k8sManager/isReady']);
+const containers = computed(() => store.state['container-engine'].containers);
+
+// Computed properties
+const containerId = computed(() => route.params.id as string || '');
+
+const currentContainer = computed((): Container | null => {
+  if (!containers.value || !containerId.value) {
+    return null;
+  }
+  return containers.value[containerId.value] || null;
+});
+
+const containerName = computed(() => {
+  if (!currentContainer.value) {
+    return containerId.value.substring(0, 12);
+  }
+  const name = currentContainer.value.containerName;
+  return name.replace(/^\//, '').replace(/_[a-z0-9-]{36}_[0-9]+/, '') || containerId.value.substring(0, 12);
+});
+
+const containerState = computed(() => {
+  if (!currentContainer.value) {
+    return 'unknown';
+  }
+  return currentContainer.value.state || currentContainer.value.status || 'unknown';
+});
+
+const isRunning = computed(() => {
+  if (!currentContainer.value) {
+    return false;
+  }
+  return currentContainer.value.state === 'running' || currentContainer.value.status === 'Up';
+});
+
+// Watchers
+watch(containerName, (name) => {
+  store.dispatch('page/setHeader', {
+    title:       name || 'Container Info',
+    description: '',
+    action:      'ContainerStatusBadge',
+  });
+}, { immediate: true });
+
+// Methods as functions
+const subscribe = async() => {
+  if (subscribeTimer.value) {
+    clearTimeout(subscribeTimer.value);
+  }
+  try {
+    if (!window.ddClient || !isK8sReady.value || !settings.value) {
+      subscribeTimer.value = setTimeout(() => subscribe(), 1_000);
```

Comment:
```suggestion
      subscribeTimer.value = setTimeout(subscribe, 1_000);
```
There is no need to preserve `this` here.  Also, `subscribe` is bound (because it's an arrow function), so the implicitly passed in `this` gets ignored anyway.